### PR TITLE
#161967022 List Users and Profiles

### PR DESCRIPTION
**Pull request by EmmanuelBeja**
#### What does this PR do?
This PR lists users and their profiles
#### Description of Task to be completed?
With this PR:
- a list of all user details is listed
- a list of all user profiles is listed
- a specific user's profile is listed
#### How should this be manually tested?
It can be tested on `Postman` as below:
1. Run the following commands in `Terminal`
```
git clone git@github.com:andela/ah-jumanji.git
git checkout ft-list-users-and-profile-161967022
virtualenv venv && source venv/bin/activate
pip install -r requirements.txt
echo "DATABASE_URL=postgres://{postgres-db-username}:{postgres-db-password}@localhost/ah_jumanji" > .env
python manage.py makemigrations && python manage.py migrate
python manage.py runserver
```
2. Send a `POST {your-server-root}:8000/api/users/register` request with a payload similar to:
```
{
  "user":{
    "username": "Aquaman",
    "email": "aquaman@marvel.com",
    "password": "theaquaman"
  }
}
```
to register a `User` and generate an auth token
3. With the auth token generated, send a `GET {your-server-root}:8000/api/users/users/`  request to view the all users.
4. To list all user profiles pass the auth token in the headers, and send a  `GET {your-server-root}:8000/api/profiles/`
5. To get a specific user profile pass the auth token in the headers, and send a `GET {your-server-root}:8000/api/profiles/Aquaman`
#### What are the relevant pivotal tracker stories?
[#161967022](https://www.pivotaltracker.com/story/show/161967022)
#### Screenshots (if appropriate)
`1. Register a User - POST {your-server-root}:8000/api/users/register`
screen shot 2018-12-06 at 12 11 21

`2. View all users - GET {your-server-root}:8000/api/users/`
screen shot 2018-12-06 at 12 16 23

`3. View all profiles - GET {your-server-root}:8000/api/users/profiles/`
screen shot 2018-12-06 at 12 12 36

`4. View specific user profile - GET {your-server-root}:8000/api/users/profiles/Aquaman`
screen shot 2018-12-06 at 12 13 51

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: authors/apps/profiles/models.py [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+    country = models.CharField(max_length=3, blank=True)
+    phone_number = models.CharField(max_length=15, blank=True)
+
+    def __str__(self):
+        """
+            Schema for representation of a Profile object in Terminal
+        """
+        return self.email
+
+    def update_profile(self, **kwargs):
```
Comment (ghost) [inline authors/apps/profiles/models.py]: This `update_profile` method can be removed. It doesn't serve any purpose

**Review by ghost — APPROVED** [review]
This passes all the tests and style checks. It can be merged.
`Clean Up` - Remove unnecessary print statements within the code (`authentication/views.py`)

File: authors/apps/profiles/models.py (hunk 1 above)
Comment (EmmanuelBeja) [reply authors/apps/profiles/models.py]: on it

**Review by GransonO — APPROVED** [review]
This is good, the hound is happy, and so are we, **`LGTM`**

**Review by gwako94 — APPROVED** [review]
Looks good to merge.

**Pull request merged**